media-sound/mixxx/files/mixxx-2.3.1-local-manuals.patch

```diff
@@ -0,0 +1,20 @@
+--- a/src/widget/wmainmenubar.cpp
++++ b/src/widget/wmainmenubar.cpp
+@@ -556,7 +556,7 @@ void WMainMenuBar::initialize() {
+     pHelpMenu->addAction(pHelpSupport);
+ 
+     // User Manual
+-    QUrl manualUrl = documentationUrl(m_pConfig->getResourcePath(),
++    QUrl manualUrl = documentationUrl("/usr/share/doc/mixxx-2.3.1",
```

Comment:
can't we update the resource path itself instead of hardcoding these occurrences?

Reply:
I confirm we can't mess with the resource path. There's even a Mixxx command line argument for setting the resource path, which is where e.g. controller mappings reside.

By the way, this was the only request from the Mixxx side (upstream), that the manual PDF would properly open through the GUI